components/InteractionDetailsItem/InteractionDetailsItem.js

```diff
@@ -0,0 +1,18 @@
+import Image from "next/image";
+import {
+  DataContainer,
+  StyledDetailsItem,
+} from "./InteractionDetailsItem.styled";
+
+export default function InteractionDetailsItem({ icon, iconAlt = "", value }) {
+  return (
+    <StyledDetailsItem>
+      <div>
+        <Image src={icon} alt={iconAlt} width={20} height={20} />
+      </div>
+      <DataContainer>
+        <div>{value}</div>
```

Comment:
Everywhere hiding these small div elements. 😁

Reply:
I wanted to keep the composition of the component very close to the ContactDetailsItem as they are almost identical - the InteractionDetailsItem just does not have the DataType div. I still need a div though because of the surrounding flexbox but not two of them in this component so I deleted the DataContainer (which was a flexbox itself just in use at the ContactDetailsItem.js).